### PR TITLE
log-conventions, 11.8 hull-node

### PR DESCRIPTION
**Pull request by mkawalek**
* connector use firehose
* logs follow log-convention document
* hull-node upgrade to 11.8 ver

### CODE REVIEW

File: server/lib/update-user.js [hunk 1]
```diff
@@ -34,18 +34,18 @@ export default function updateUser({ metric, ship, client, isBatch = false }: an
   }
 
   if (!synchronized_segments.length) {
-    hull.logger.info('outgoing.user.skip', _.merge(
-      _.pick(user, "id", "external_id", "email"),
-      { reason: 'No Segments configured. All Users will be skipped' }
-    ));
+    hull.logger.info('outgoing.user.skip', {
+      userIdent: { email: user.email, external_id: user.external_id, hull_id: user.id },
```
Comment (michaloo) [inline server/lib/update-user.js]: Same thing as for hubspot @mkawalek -> we need to use `hull.asUser().logger.info()` here

File: server/lib/webhook.js [hunk 2]
```diff
@@ -27,10 +28,10 @@ export default function webhook({ webhooks_urls, hull, payload = {} }: any) {
         // Something happened in setting up the request that triggered an Error
         hull.logger.debug('webhook.error', { message: msg });
       }
-      hull.logger.info('outgoing.user.error', _.merge(
-        _.pick(payload.user, "id", "external_id", "email"),
-        { errors }
-      ));
+      hull.logger.error('outgoing.user.error', {
+        userIdent: { email: payload.user.email, external_id: payload.user.external_id, hull_id: payload.user.id },
```
Comment (michaloo) [inline server/lib/webhook.js]: @mkawalek let's remove `userIdent`

File: server/lib/webhook.js [hunk 3]
```diff
@@ -6,7 +6,8 @@ export default function webhook({ webhooks_urls, hull, payload = {} }: any) {
   return _.map(webhooks_urls, url => axios.post(url, payload)
     .then(
       ({ data, status, statusText }) => {
-        hull.logger.info('outgoing.user.success', _.pick(payload.user, "id", "external_id", "email"));
+        // hull.asUser({ email: email.payload.user.email, external_id: payload.user.external_id, id: payload.user.id }).logger.info('outgoing.user.success');
```
Comment (michaloo) [inline server/lib/webhook.js]: lets remove the commented line here.

File: server/lib/webhook.js [hunk 4]
```diff
@@ -27,10 +27,10 @@ export default function webhook({ webhooks_urls, hull, payload = {} }: any) {
         // Something happened in setting up the request that triggered an Error
         hull.logger.debug('webhook.error', { message: msg });
       }
-      hull.logger.info('outgoing.user.error', _.merge(
-        _.pick(payload.user, "id", "external_id", "email"),
-        { errors }
-      ));
+      hull.logger.error('outgoing.user.error', {
+        email: payload.user.email, external_id: payload.user.external_id, hull_id: payload.user.id,
```
Comment (sbellity) [inline server/lib/webhook.js]: Let's use the `asUser` method to get a scoped logger here too

**Pull request merged**